test/cmdlineTests/viair_subobject_optimization/output

```diff
@@ -134,7 +126,9 @@ sub_0: assembly {
       tag_4
       jumpi
       pop
```

Comment:
This could be removed as well :)

Reply:
Yep, I know - not entirely sure why it isn't yet. Either the logic in the rest of the PR misses some cases or this only comes into existence during evmasm optimization...